databricks/koalas/frame.py

```diff
@@ -50,7 +50,6 @@
 from databricks.koalas.utils import scol_for
 from databricks.koalas.typedef import as_spark_type
 
-
```

Comment:
@RainFung, let's avoid to add unrelated changes next time. It makes backporting or reverting more difficult in the future.

Reply:
@RainFung, please don't resolve a comment when it's unresolved.